src/services/ui/e2e/tests/home/index.spec.ts

```diff
@@ -1,21 +1,33 @@
 import { test, expect } from "@playwright/test";
-import * as $ from "@/selectors";
+
+const password = process.env.VITE_BOOTSTRAP_USERS_PW!;
 
 test("has title", async ({ page }) => {
   await page.goto("/");
-
-  // Expect a title "to contain" a substring.
   await expect(page).toHaveTitle(/CMS MAKO/);
 });
 
-test("get issues link", async ({ page }) => {
-  const navSelectors = new $.NavSelectors(page);
+test("has faq Page", async ({ page }) => {
+  await page.goto("/");
+  await page.getByRole("link", { name: "FAQ" }).click();
+});
+
+test("log in test", async ({ page }) => {
   await page.goto("/");
+  await page.getByRole("button", { name: "Sign In" }).click();
+  await page.getByRole("textbox", { name: "name@host.com" }).click();
+  await page.getByRole("textbox", { name: "name@host.com" }).fill("george@example.com");
+  await page.getByRole("textbox", { name: "Password" }).click();
+  await page.getByRole("textbox", { name: "Password" }).fill(password);
+  await page.getByRole("button", { name: "submit" }).click();
 
-  // Click the issues link.
-  await navSelectors.issuesDropDown.click();
-  await navSelectors.allIssuesLink.click();
+  const isLoggedIn = await page.getByRole("link", { name: "Dashboard" }).isVisible();
+  const isFailed = await page.getByRole("button", { name: "Sign Out" }).isVisible();
 
-  // Expects the URL to contain intro.
-  await expect(page).toHaveURL(/.*issues/);
-});
+  if (isLoggedIn) {
+    expect(isFailed).toBeTruthy();
+  }
+  else {
+    expect(isFailed).toBeFalsy();
+  }
```

Comment:
I think we should extract the isFailed logic into its own test case. "Log in should fail test" and use the isFailedLogic and use a username/password combo that will fail and expect it to fail.

Reply:
We can discuss this on our call tomorrow. I don't think it is best to intentionally set up the logic to fail.

Reply:
yeah i dont seee what this test is doing, is it suppose to be loggin the user in? if so lets expect that it does and check that it logs in.